### PR TITLE
[Manual Search] Fixed saving status of history table

### PR DESCRIPTION
**Pull request by p0psicles**
* Fixes the automatic hiding of table when results are refreshed. Now it saves the status. If user toggles the history page to show, it keeps showing.

### CODE REVIEW

File: gui/slick/js/core.js [hunk 1]
```diff
@@ -2643,11 +2655,13 @@ var SICKRAGE = {
             // Moved and rewritten this from displayShow. This changes the button when clicked for collapsing/expanding the
             // "Show History" button to show or hide the snatch/download/failed history for a manual searched episode or pack.
             $(function() {
-                $('.collapse.toggle').on('hide.bs.collapse', function () {
+                $('body').on('hide.bs.collapse', '.collapse.toggle', function () {
```
Comment (p0psicles) [inline gui/slick/js/core.js]: I started with binding these to 'body', but maybe that wasn't needed in hindsight.

File: gui/slick/js/core.js [hunk 2]
```diff
@@ -2505,13 +2505,24 @@ var SICKRAGE = {
...
+            }
+
+            $.fn.loadContainer = function(path, loadingTxt, errorTxt, callback) {
                 updateSpinner(spinner, loadingTxt);
                 $(this).load(srRoot + path + ' #container', function(response, status) {
                     if (status === "error") {
                         updateSpinner(spinner, errorTxt, false);
                     }
+                    if (typeof callback !== 'undefined') {
+                        callback();
```
Comment (p0psicles) [inline gui/slick/js/core.js]: I've implemented it as a callback. But that was to be able to use the loadContainer() function genericly, without requiring to pass a callback function, although it's only used by one caller.

File: gui/slick/views/snatchSelection.mako [hunk 3]
```diff
@@ -187,7 +187,7 @@
     <div id="searchNotification"></div>
 
     <div class="clearfix"></div>
-    <div id="wrapper">
+    <div id="wrapper" data-history-toggle="hide">
```
Comment (p0psicles) [inline gui/slick/views/snatchSelection.mako]: This is the meat of the PR. Here I make sure the last status is saved as a data attribute.

**Comment by fernandog** [comment]
Approved

**Comment by fernandog** [comment]
@medariox confirm it fixed?

**Comment by medariox** [comment]
Works perfectly. Thanks @p0psicles

**Comment by fernandog** [comment]
rebased

File: gui/slick/js/core.js (hunk 1 above)
Comment (OmgImAlexis) [inline gui/slick/js/core.js]: That most probably is needed since it'll allow the bind to hold even if you remove and/or add in new items using that class.

**Comment by p0psicles** [comment]
Approved

**Pull request merged**